ServerCore/Pages/Teams/Answers.cshtml

```diff
@@ -43,7 +43,7 @@
         {
         <tr>
             <td>
-                @item.TimeSubmitted
+                <time>@item.TimeSubmitted.ToString("yyyy-MM-ddTHH:mm:ssZ")</time>
```

Comment:
Any chance the format string could be part of the function so we don't have a copy-pasted string in quite so many places?